README.md

```diff
@@ -88,3 +88,33 @@ After you've started the dev server, you can open the interactive browser tests
 ```sh
 npm run cypress:open
 ```
+
+### Chore: Dependency Updates
+
+You can see which dependencies have new releases by first making sure your local dependencies are up-to-date by executing `npm ci` and then running `npm outdated`.
+The following dependencies should be ignored:
+
+- Wikit (i.e. `@wmde/wikit-tokens` and `@wmde/wikit-vue-components`):
+  we’re using a newer pre-release version and don’t want to downgrade to the latest full release.
+- Vue and Vuex:
+  in production, we use the versions shipped by MediaWiki core,
+  so we should use the same versions for testing.
+
+All other dependencies should generally be updated to the latest version.
+If you discover that a dependency cannot be updated for some reason, please add it to the above list.
+If a dependency can only be updated with substantial manual work,
+you can create a new task for it and skip it in the context of the current chore.
+
+The recommended way to update dependencies is to collect related dependency updates into grouped commit.
+This can be one commit for each of:
```

Comment:
I would like to weaken this list a bit, as many variations of that might make sense.

```suggestion
For example, this can be one commit for each of:
```

Reply:
Well, it was “for example” in the task description I based it on, but then I saw that the previous chore seemed to have followed this pattern fairly closely, so it felt strange to only call it an example… but I think that might sound better with the other change (to explain why this is the recommended way), so okay.